release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -727,6 +727,12 @@ service class EchoService {
 }
 ```
 
+- `tcp:ListenerConfig` and `tcp:ClientConfig` renamed to `tcp:ListenerConfiguration` and `tcp:ClientConfiguration`
```

Comment:
Shall we say 
```suggestion
- Renamed `tcp:ListenerConfig` and `tcp:ClientConfig` to `tcp:ListenerConfiguration` and `tcp:ClientConfiguration`
```
Because in other places we have followed that pattern.

Reply:
updated